src/hibernate-resume/hibernate-resume-generator.c

```diff
@@ -214,6 +215,10 @@ static int process_resume(void) {
         if (r < 0)
                 log_warning_errno(r, "Failed to write device timeout drop-in, ignoring: %m");
 
+        device_escaped = cescape(arg_resume_device);
```

Comment:
hmm, this looks incomplete. You need specifier escaping too, i.e. specifier_escape()

Reply:
-> #28817